### PR TITLE
Add a11y logic to topic page

### LINKED ISSUE #9969: Implement Screen Reader UX to Topic Page
**Is your feature request related to a problem? Please describe.**
Topic page UX should follow screen reader UX specifications as shown here: https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/61e7dcc2f7ddc1894e3c4a7a

Example topic page: https://www.test.bbc.com/pidgin/new_topics/cqywjyzk2vyt

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves #9969

**Overall change:**
Updates Topic Page to follow Screen Reader UX.

**Code changes:**

- Adds main element with a role of main to the content of Topic page
- Switches tags in promo around so that the a tag is within the h2
- Adds logic for if multiple promos exist then use ul/li.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by Louis-Matsika — APPROVED** [review]

**Review by DarioR01 — APPROVED** [review]

File: src/app/pages/TopicPage/TopicGrid/index.jsx [hunk 1]
```diff
@@ -52,13 +59,23 @@ const Wrapper = styled.div``;
...
   const { dir } = useContext(ServiceContext);
+  const hasMultiplePromos = promos.length > 1;
+  const firstPromo = promos[0];
   return (
     <Wrapper>
-      {promos.map(promo => (
-        <Item key={promo.id} dir={dir}>
-          <TopicPromo {...promo} />
+      {hasMultiplePromos ? (
+        <TopicList role="list">
```
Comment (DarioR01) [inline src/app/pages/TopicPage/TopicGrid/index.jsx:67]: Should we be adding unit tests for both these paths?

Comment (nicola-d-bbc) [reply src/app/pages/TopicPage/TopicGrid/index.jsx:67]: can do. i've just been testing manually but can also add in unit tests as the other pages have that have the same sort of logic :)

**Review by ghost — APPROVED** [review]
Happy for this to be merged once ul/li tests are added 😃

File: src/app/pages/TopicPage/TopicPage.jsx [hunk 2]
```diff
@@ -27,24 +27,26 @@ const TopicPage = ({ pageData }) => {
...
-        type="CollectionPage"
-        seoTitle={title}
-        headline={title}
-        entities={promoEntities}
-      />
-      <TopicTitle>{title}</TopicTitle>
-      <TopicGrid promos={promos} />
-    </Wrapper>
+    <main role="main">
+      <Wrapper>
```
Comment (jroebu14) [inline src/app/pages/TopicPage/TopicPage.jsx]: Can we just make this the main element with the role? It would remove an unnecessary level in the DOM tree.

**Review by jroebu14 — APPROVED** [review]
Great work 👍

**Pull request merged**